app/AccountancyModule/CampModule/templates/Participant/default.latte

```diff
@@ -1,19 +1,17 @@
 {block #title}{$event->getDisplayName()} - seznam účastníků{/block}
 
 {block #content}
-{include ../ActionMenu.latte}
-<h2>{$event->getDisplayName()}</h2>
-<div class="row" n:if="$missingAvailableAutoComputed">
-    <div class="coll-md-6 alert alert-info">
-        Není aktivní dopočítávání osob! <a n:href="activateAutocomputedParticipants! $aid" class="btn btn-primary">Aktivovat</a>
-    </div>
+{include ../header.latte}
+
+<div class="alert alert-warning text-center" n:if="$missingAvailableAutoComputed">
+    Není aktivní automatické dopočítávání osobodnů a dětodnů ve Skautisu. <a n:href="activateAutocomputedParticipants! $aid" class="btn btn-primary ml-2">Aktivovat</a>
 </div>
 
 <div class="row">
-    <div class="col-sm-5" n:inner-if="$canAddParticipants">
```

Comment:
zkoušel jsi hranici md místo lg?

Reply:
Jj, těsně na hraně už se to rozyspe.
![image](https://user-images.githubusercontent.com/5658260/79768414-0ccfd180-832b-11ea-8bd3-8b17e554aabb.png)

Reply:
:/